src/main/scala/tech/cryptonomic/conseil/tezos/Fees.scala

```diff
@@ -0,0 +1,23 @@
+package tech.cryptonomic.conseil.tezos
+
+object Fees {
+
+  type OperationKind = String
+
+  /**
+    * Representation of estimation of average fees for a given operation kind.
+    * @param low       Medium - one standard deviation
+    * @param medium    The mean of fees on a given operation kind
+    * @param high      Medium + one standard deviation
+    * @param timestamp The timestamp when the calculation took place
+    * @param kind      The kind of operation being averaged over
+    */
+  final case class AverageFees(
+      low: Int,
+      medium: Int,
+      high: Int,
+      timestamp: java.sql.Timestamp,
```

Comment:
I would migrate it to `java.time.Instant`. That class is mutable and has inconsistent API.

Reply:
Good one, but it would need a clear semantic of how that translates to db values... I've been hurt by timezone and timestamps before with databases, so I want to be sure that we store what we mean to.
I'd love to hear from your experience on nautilus (we can take it on the chat)...

Why not open a new issue, though?